ext/compressors/zlib/zlib_compress.c

```diff
@@ -32,18 +32,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include <wiredtiger_config.h>
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-/*
- * We need to include the configuration file to detect whether this extension
```

Comment:
Now I understand. Some of the compression engines had the comment, some didn't, so I just dropped it. I've added it back in for all of the engines.

I didn't chase the reason for the Windows-specific define. Zlib was the only engine that had it, and both snappy and lz4 used `inline`, so I was reasonably sure it wasn't necessary.